internal/testsetup/testsetup.go

```diff
@@ -55,6 +55,13 @@ type testAzureSubscription struct {
 	Exocompute struct {
 		SubnetID string `json:"subnetId"`
 	} `json:"exocompute"`
+
+	// should be in EastUS2 region.
```

Comment:
What does this requirement mean in practice? Should it be something that we check during validation of the resource group?

Reply:
Have added more comments.
This is basically where managed identity should be created for integration tests.

Reply:
Ah I see, it's because we have hardcoded "eastus2" in the actual tests? The comment make sense. We should probably add something similar for our code. :slightly_smiling_face:

Reply:
Not in actual tests but in integration tests here.
I have used the same pattern that was already there in `polaris_test.go`

Reply:
:+1: